docs/lib/usage/starlite-app/app-hooks.rst

```diff
@@ -0,0 +1,166 @@
+ASGI Events and Application Hooks
+===============================
+
+In the previous introductory section of the documentations we briefly mentioned about the root
+:class:`Starlite <starlite.app.Starlite>` class. This class accepts several `optional` arguments
+and a couple of those arguments can be configured to control the behavior of your Starlite application
+based on certain ASGI events. This section of the documentations take a deeper dive into how you
+can use those hooks to configure your application throughout its lifespan.
+
+Brief Overview of Application Hooks and Their Respective Events
+-------------------------------------------------------------
+
+All ASGI webservers (like `Uvicorn <https://www.uvicorn.org>`_ or
+`Daphne <https://github.com/django/daphne>`_ and such) emits certain events through the lifespan of the
+application they are invoked with. And Starlite can hook into those events to perform certain tasks as
```

Comment:
this is not english `application they are invoked with`. Rephrase this please.

`And Starlite can hook into those events to perform certain tasks as`

those -> these.
And -> should not be the beginning of a sentence
`And Starlite` -> hanging sentence.

Rewrite this please